src/main/webapp/app/overview/course-card.component.ts

```diff
@@ -45,7 +46,7 @@ export class CourseCardComponent implements OnChanges {
         name: 'vivid',
         selectable: true,
         group: ScaleType.Ordinal,
-        domain: ['#32cd32', '#ff0000'], // colors: green, red
+        domain: [GraphColors.GREEN, GraphColors.RED], // colors: green, red
```

Comment:
```suggestion
        domain: [GraphColors.GREEN, GraphColors.RED],
```

Reply:
I changed the color implementation in `doughnut-chart.component.ts `accordingly in 37bdc4e.